### DISCUSSION TITLE
What does SyncUp mean

### DISCUSSION THREAD

**Discussion by danie-dejager** [zrepl/zrepl#830] [Q&A] [answered]
I search the documentation and did not find what `SyncUp` means. I have the following status:
```
Job: snapjob                                 
Type: snap                                   
                                             
Pruning snapshots:                           
   ...                                       
                                             
Snapshotting:                                
   Type: periodic                            
   Status: SyncUp                            
   Sleep until: 2024-05-16 11:59:27 +0100 BST
```
Stuck on that for a couple of hours now.

**Comment by problame** [comment, marked as answer]
Sorry for the late reply.

---

The snapshotter waits for all `filesystem`s' latest snapshot matching `prefix` to be at least `period` old.

See here: https://zrepl.github.io/configuration/snapshotting.html#periodic-snapshotting
